test/e2e/deployment.go

```diff
@@ -204,7 +270,33 @@ func testRollingUpdateDeploymentEvents(f *Framework) {
 	// Create a deployment to delete nginx pods and instead bring up redis pods.
 	deploymentName := "redis-deployment-2"
 	Logf("Creating deployment %s", deploymentName)
-	_, err = c.Deployments(ns).Create(newDeployment(deploymentName, replicas, deploymentPodLabels, "redis", "redis"))
+	newDeployment := extensions.Deployment{
```

Comment:
Use the newDeployment method.
Add another param there for deploymentStrategyType.